packages/benchmarks/src/implementations/styled-jsx/Box.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable react/prop-types */
+import classnames from 'classnames';
+import React from 'react';
+import View from './View';
+
+const getColor = color => {
+  switch (color) {
+    case 0:
+      return '#14171A';
+    case 1:
+      return '#AAB8C2';
+    case 2:
+      return '#E6ECF0';
+    case 3:
+      return '#FFAD1F';
+    case 4:
+      return '#F45D22';
+    case 5:
+      return '#E0245E';
+    default:
+      return 'transparent';
+  }
+};
+
+const Box = props => {
+  const { className, children: styles } = (
+    <scope>
```

Comment:
styled-jsx doesn't allow to style components (`View`) or pass styles down, the only way to style descendants is this or `:global()`. This workaround might add overhead. Would you accept an implementation of the benchmark that doesn't make use of a `View` component?